tapdb/sqlc/queries/universe.sql

```diff
@@ -113,14 +113,18 @@ SELECT * FROM universe_servers;
 -- name: InsertNewSyncEvent :exec
 WITH group_key_root_id AS (
     SELECT id
-    FROM universe_roots
+    FROM universe_roots roots
     WHERE group_key = @group_key_x_only
+      AND roots.proof_type = @proof_type
 ), asset_id_root_id AS (
     SELECT leaves.universe_root_id AS id
     FROM universe_leaves leaves
+    JOIN universe_roots roots
```

Comment:
Ah actually a bug fix? Or I guess we'd still have the same return value, this join is just slightly more correct.

Reply:
This additional `join` ensures that we select the asset ID from the row which has the correct proof type, but functionally it makes no difference.